apps/builddao/widget/Compose.jsx

```diff
@@ -1,10 +1,11 @@
-const { Avatar, Button } = VM.require("buildhub.near/widget/components");
+const { Avatar, Button, InputField, TextEditor } = VM.require(
+  "buildhub.near/widget/components"
+);
 
-Avatar = Avatar || (() => <></>);
 Button = Button || (() => <></>);
 
-const draftKey = props.feed.name || "draft";
-const draft = Storage.privateGet(draftKey);
+// const draftKey = props.feed.name || "draft";
```

Comment:
Now that template toggling is working w/ confirmation screen, can you uncomment this so I can have my post saved to draft?